### PR TITLE
Reorganise test.json, add new "TerriaJS test data" section

### PR DESCRIPTION
**Pull request by stevage**

### CODE REVIEW

File: wwwroot/init/test.json [hunk 1]
```diff
@@ -1,278 +1,355 @@
...
+            },
+            {
+              "name": "Airports (OGR->GeoJSON, zipped)",
+              "type": "ogr",
+              "url": "test/airports.zip"
+            },
+            {
+              "name": "Styling test",
+              "type": "geojson",
+              "url": "test/test-styles.geojson"
```
Comment (kring) [inline wwwroot/init/test.json]: This file doesn't exist.

File: wwwroot/init/test.json [hunk 2]
```diff
@@ -1,278 +1,355 @@
...
                 },
                 {
-                    "name": "Western Australia",
-                    "url": "http://www2.landgate.wa.gov.au/ows/wfspublic_4326/wfs",
-                    "type": "wfs-getCapabilities"
+                  "position": {
+                    "cartographicDegrees": [
+                      133,
+                      -25,
+                      ""
```
Comment (kring) [inline wwwroot/init/test.json]: This catalog item doesn't work anymore because zeros were replaced with empty strings.

**Comment by kring** [comment]
2011 Census AUST (STE) and State Mask Test also don't work, but that's not a result of this change.

**Comment by kgrochow** [comment]
Both will be fixed when I check in the abs/region changes.  States needs to support both names and codes.  The tests depends on codes, current default is names - well, name abbreviations.

**Pull request merged**